protocol/src/main/java/io/camunda/zeebe/protocol/ZbColumnFamilies.java

```diff
@@ -1,11 +1,19 @@
 /*
- * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
- * one or more contributor license agreements. See the NOTICE file distributed
- * with this work for additional information regarding copyright ownership.
- * Licensed under the Zeebe Community License 1.1. You may not use this file
- * except in compliance with the Zeebe Community License 1.1.
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
  */
-package io.camunda.zeebe.engine.state;
+package io.camunda.zeebe.protocol;
```

Comment:
:thought_balloon: Yes, definitely, this should be moved somewhere else :smile:

Reply:
Do you already have an idea?

Reply:
OTOH, it wouldn't be that crazy for the whole DB persistence layer to have its own module, e.g. `engine-state` or something, then that could go in there. I guess not something we can unilaterally decide. Or it can go in the API module once we have one? Since that will be a dependency for both the stream platform impl and the engine?

I guess in the stream platform module we just need it for the processed position and such things? We don't really access the engine states, right? It would be cool if we could somehow share the DB but not know about the engine state/column families either...like a way to extend the state? We open the DB, create our own state/columns, then pass it on to the engine. They also don't need to know about our own columns/states. Not sure how feasible that is due to how we index though :(

Reply:
Yeah state module is something I also wanted since a while but still this wouldn't solve it since stream platform doesn't want to know all these details.

What I thought first I could extend an enum, first define the two columns and then later extend in the engine with more columns, but unfortunately enums can't be extended since they are final :sweat_smile: 

Theoretically, we could do something similar via composition I guess, we have some default columns defined on base level which always need to be defined. Call the class BaseColumns these are used by stream-platform, and engine defines own columns and also uses BaseColumns. :shrug: But I felt this is too much for this PR. :)

Reply:
Yeah that's fair. I think we should do it long term (both the state module and taking the composition approach, so each module doesn't need to know all columns).